package.json

```diff
@@ -29,6 +29,7 @@
   "scripts": {
     "start": "(cd examples/main && (if [ ! -d 'node_modules' ]; then npm i; fi) && npm run start-local)",
     "build": "./scripts/build.sh && ./scripts/collect-metrics.sh",
+    "flow": "flow check",
```

Comment:
I recommend to use another name like `typecheck`. Running `yarn flow` in development process will take too much time with this script. The original one runs the server.